pkg/console/operator/sync_v400.go

```diff
@@ -47,76 +47,69 @@ func (co *consoleOperator) sync_v400(updatedOperatorConfig *operatorv1.Console,
 	toUpdate := false
 
 	rt, rtChanged, rtErr := co.SyncRoute(set.Operator)
+	toUpdate = toUpdate || rtChanged
+	co.HandleProgressing(updatedOperatorConfig, "ConsoleRouteSync", rtErr)
```

Comment:
`HandleProgressing()` can't be hidden behind the `if` as we need it to toggle `true||false` based on the error so we don't end up in a permanent `Progressing` situation.  Same as `Degraded`.

Fortunately our e2e test should ensure we don't get in this state.  HOWEVER, if we ever remove a condition, this could get interesting between versions during an upgrade.